src/wakatimeScript.ts

```diff
@@ -243,21 +243,50 @@ const recordHeartbeat = async (apiKey: string, payload: Record<string, unknown>)
   }
 };
 
+const parseCanva = () => {
+  const canvaProject = document.getElementsByClassName('rF765A');
+  if (canvaProject.length === 0) return;
+
+  const projectName = (document.head.querySelector('meta[property="og:title"]') as HTMLMetaElement)
+    .content;
+  return {
+    editor: 'Canva',
+    language: 'Canva Design',
+    project: projectName,
+  };
+};
+
+const parseFigma = () => {
+  const projectName = (document.querySelector('span[data-testid="filename"]') as HTMLElement)
+    .innerText;
+  return {
+    editor: 'Figma',
+    language: 'Figma Design',
+    project: projectName,
+  };
+};
```

Comment:
Thanks for your contribution.
Here we are missing a check to only send heartbeats when a user has a project open
something like [this](https://github.com/wakatime/chrome-wakatime/pull/185/files#diff-5fe6967e0cf99e4aad7de15da29ed8cf11e005156620e5a399cb0ee243b9fe66R260)

```
  const figmaProject = document.getElementsByClassName('gpu-view-content');
  if (figmaProject.length === 0) return;
```

Reply:
Right, fixed. Thank you for review.